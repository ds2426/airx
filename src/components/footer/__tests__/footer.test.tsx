import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../index';
import { expect, test } from 'vitest';

function toJson(component: renderer.ReactTestRenderer) {
  const result = component.toJSON();
  expect(result).toBeDefined();
  return result as renderer.ReactTestRendererJSON;
}

test('Footer renders', () => {
  const component = renderer.create(<Footer />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
