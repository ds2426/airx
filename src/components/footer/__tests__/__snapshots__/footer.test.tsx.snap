// Vitest Snapshot v1

exports[`Footer renders 1`] = `
<div
  className="MuiBox-root css-v63k1i"
>
  <span>
    Â© 2022
  </span>
</div>
`;
